### PR TITLE
Display gamelist.xml including image on rom list page

### PR DESCRIPTION
**Pull request by kdorff**
I wanted some of the data from gamelist.xml scraping in your rom list so I've changed the code to do that.

How my change looks: http://imgur.com/a/e2rI0 

I wanted to gauge your interest in this. I don't think this adds much overhead to page generation time (parsing the XML, looking the game up in the dictionary), although page render time (and render size) will be increased because it is sending images and a lot more information.

Note: I am neither a Python nor Django programmer. I haven't fully run through every scenario but am pretty happy this seems to work for me in my configuration with RetroPie (recent install).  If you can use any of this but do it in a better way, feel free. My feelings won't be hurt - I just want the data on the page :)

I'm happy to make changes to this PR (if I can figure out what you want changed) or do a bit more testing if this interests you, too, esp if you can specify some scenarios.

I noticed you placed the .pyc files in github which I don't think makes sense. I removed them here and added a .gitignore. You might want to do that as a separate commit, perhaps?

Thoughts?

### CODE REVIEW

**Comment by kdorff** [comment]
I have some ideas on making the table fully searchable - which would be really nice, too.

**Comment by meleu** [comment]
Hi @kdorff , unfortunatelly @botolo78 seems to be busy or lost interest in this project (I hope nothing serious has happened to him). His last post on RetroPie forum and his last activity here on github were in July 2016.

He gave me access to this repo when it started because I wrote the [rpmanager.sh script](https://github.com/botolo78/RetroPie-Manager/blob/retropie/rpmanager.sh) but I'm unable to code Python/Django/Web-things...

I could merge your PR, but I'm not sure if the added overhead would be acceptable on a raspi1 or raspi0. Could you implement this as an option?

> I noticed you placed the .pyc files in github which I don't think makes sense. I removed them here and added a .gitignore. You might want to do that as a separate commit, perhaps?

Yeah. A PR removing unnecessary files and a better .gitignore would be appreciated.

**Comment by kdorff** [comment]
TL;DR: I didn't realize this project was essentially dead. I'll give my contribution some more thought or just start a new app since I don't know Django and am not a Python developer (meaning: neither are us are fit or interested in maintaining this project but it's mostly dead, otherwise).

Hi @meleu that is all very unfortunate news. While Django/Python isn't my expertise, this project was very close to what I wanted: a web-based ROM viewer to run on my RetroPie to help me validate the list of ROMs and how they have been scraped. I thought the learning curve (Django) might be worth it. I saw the project had recent contributions, I didn't realize it was so dead. My failure for not researching deeply enough.

The deadness of the project makes me re-think contributions. My current thinking would be to re-write as a simpler RetroPie (Emulation station) ROM manager

* list of systems
* For a selected system: 
   * list with scraper details, images
   * Dynamic filtering
   * Ability to delete ROMs
   * POSSIBLY upload (I don't really care about this)

My primary skill is Groovy/Grails and the code would be MUCH simpler than this project to both write, follow, understand, test, and maintain. This is likely the direction I will go.

As I've only been playing with RetroPie for a couple weeks, I don't understand how I'll tie this all into the RetroPie-setup, make it auto-start, etc. but I can learn those if I get far enough into the project.

**Comment by meleu** [comment]
>  My current thinking would be to re-write as a simpler RetroPie (Emulation station) ROM manager

I see a bunch of similar projects in [RetroPie forums](https://retropie.org.uk/forum/unr). Maybe you can choose one to contribute (or start your own from scratch).

Recently I was impressed by this one: https://retropie.org.uk/forum/topic/10164/web-app-wip-please-give-it-a-name

> I don't understand how I'll tie this all into the RetroPie-setup, make it auto-start, etc.

I can help you with it. I'm comfortable with shell scripting and have some familiarity with RetroPie script modules.
